templates/OperationSendRequestSnippet.stencil

```diff
@@ -10,7 +10,7 @@ let context = PipelineContext.of(keyValues: [
    {{ param.key }} : {{ param.value}} as AnyObject
    {% endfor %}
 ])
-if let cancellationToken = options?.cancellationToken {
+if let cancellationToken = {{ op.clientMethodOptionsViewModel.variableName }}?.cancellationToken {
```

Comment:
Again, this variable should always be called just `options` (i.e. not `listThreadOptions` or anything like that) so I'm not seeing the need for dynamically generated names here.

Reply:
Changed per feedback.